locale/en-US/add-on.properties

```diff
@@ -183,14 +184,17 @@ door_hanger_button=OK
 
 infobar_screenshare_browser_message2=You are sharing your tabs. Any tab you click on can be seen by your friends
 infobar_screenshare_paused_browser_message=Tab sharing is paused
+infobar_screenshare_stop_sharing_message=You are no longer sharing your tabs
 infobar_button_pause_label=Pause
 infobar_button_pause_accesskey=P
-infobar_button_restart_label=Restart
+infobar_button_restart_label2=Restart sharing
```

Comment:
Is "Restart" supposed to replace "Resume"? If so, we should be able to remove the Resume label/accesskey and use "R" for Restart? Also, this new label should probably be used in code somewhere.